packages/common-ts/src/base-service/base-service-v2.ts

```diff
@@ -116,6 +116,12 @@ export abstract class BaseServiceV2<
    * App server.
    */
   protected server: Server
```

Comment:
I guess we could just make this public

Reply:
It shouldn't be public because the router that is registered on `routes` is supposed to be a self contained server.  The user of base service shouldn't be able to touch the actual server object and if they do, they are likely doing something wrong.

Reply:
I don't like the practice of having functions that are specific for tests. Can `server` be immutable and then we have a getter for it?